app/services/support_interface/unexplained_breaks_in_work_history_export.rb

```diff
@@ -0,0 +1,108 @@
+module SupportInterface
+  class UnexplainedBreaksInWorkHistoryExport
+    class UnexplainedBreak
+      def initialize(month_range:)
+        @month_range = month_range
+      end
+
+      def start_date
+        @month_range.first.prev_month
+      end
+
+      def end_date
+        @month_range.last.next_month
+      end
+
+      def length
+        ((end_date.year * 12) + end_date.month) - ((start_date.year * 12) + start_date.month)
```

Comment:
This was necessary to make sure the length calculated matched up with the one shown on the work history timeline on the `support/application/{id}` page.

This logic has been taken from [work_history_item_component.rb](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/components/work_history_item_component.rb).